packer/epoch.json

```diff
@@ -0,0 +1,74 @@
+{
+  "variables": {
+    "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+    "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}"
+  },
+  "builders":
+  [
+    {
+      "type": "amazon-ebs",
+      "region": "us-west-2",
+      "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "name": "ubuntu/images/*ubuntu-xenial-16.04-amd64-server-*",
+          "root-device-type": "ebs"
+        },
+        "owners": ["099720109477"],
+        "most_recent": true
+      },
+      "instance_type": "t2.micro",
+      "ssh_username": "ubuntu",
+      "ami_name": "epoch-ubuntu-16.04-v{{timestamp}}",
```

Comment:
While AWS builder supports `force_deregister`, OS does not. Thus `{{timestamp}}` is used here for consistency.
Otherwise I'd prefer manual changing v1, v2, vX and don't waster time to cleanup old images manually.